includes/migration-guide/runtime/asp/no-longer-able-set-enableviewstatemac-false.md

```diff
@@ -2,17 +2,17 @@
 
 #### Details
 
-ASP.NET no longer allows developers to specify <code>&lt;pages enableViewStateMac=&quot;false&quot;/&gt;</code> or <code>&lt;@Page EnableViewStateMac=&quot;false&quot; %&gt;</code>. The view state message authentication code (MAC) is now enforced for all requests with embedded view state. Only apps that explicitly set the EnableViewStateMac property to <code>false</code> are affected.
+ASP.NET no longer allows developers to specify `<pages enableViewStateMac=&quot;false&quot;/>` or `<@Page EnableViewStateMac=&quot;false&quot; %>`. The view state message authentication code (MAC) is now enforced for all requests with embedded view state. Only apps that explicitly set the EnableViewStateMac property to `false` are affected.
 
 #### Suggestion
 
 EnableViewStateMac must be assumed to be true, and any resulting MAC errors must be resolved (as explained in [this guidance](https://support.microsoft.com/kb/2915218), which contains multiple resolutions depending on the specifics of what is causing MAC errors).
 
-| Name    | Value       |
-|:--------|:------------|
-| Scope   |Major|
-|Version|4.5.2|
-|Type|Runtime|
+| Name    | Value   |
```

Comment:
I hope that you used the [Evenly distribute selected table](https://review.docs.microsoft.com/en-us/help/contribute/docs-authoring/reformat-table?branch=master#evenly-distribute-selected-table) option from the DAP?!

Reply:
I thought I was using your command, but now it looks like it's from the Markdown Table extension.

![image](https://user-images.githubusercontent.com/24882762/128535034-7c2d11cd-9621-444d-81c5-5a5d44f82f54.png)